ethereum/testdata/block_0xc10a51a3898a85c7165a9d883acc9a68f139934d0cb91dfad4c7d3a7c1a1960d.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Just curious, how did you generate these test data?

Reply:
I queried the actual node to get this info. so it's real node response